src/components/DatetimeRangePicker/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { FC } from "react";
+import "react-day-picker/lib/style.css";
+import "moment/locale/de";
+import { DayPickerInput } from "@components/DayPickerInput";
+import { TimeInput } from "@components/TimeInput";
+
+export const DatetimeRangePicker: FC = () => {
+  const [fromDate, setFromDate] = useState(new Date());
+  const [toDate, setToDate] = useState(new Date());
+  const [fromTime, setFromTime] = useState("00:00");
+  const [toTime, setToTime] = useState("00:00");
+  const modifiers = { start: fromDate, end: toDate };
+
+  return (
+    <div className='flex gap-2'>
```

Comment:
I'm wondering if we could add a `flex-wrap` here to ensure that the elements don't overflow the screen horizontally on small devices?

It would require some z-index/stack context changes though:
![Screenshot 2021-08-11 at 09 48 57](https://user-images.githubusercontent.com/15640196/128990773-bd4d97ab-319f-4cf7-a362-20a0234a62f4.png)

Reply:
Thx for your review! Good point! I will fix that! :)